moveit_core/collision_detection_bullet/test/test_bullet_continuous_collision_checking.cpp

```diff
@@ -129,9 +164,9 @@ void addCollisionObjectsMesh(collision_detection_bullet::BulletCastBVHManager& c
   obj1_poses.push_back(static_box_pose);
   obj1_types.push_back(collision_detection_bullet::CollisionObjectType::USE_SHAPE_TYPE);
 
-  checker.addCollisionObject("static_box_link", collision_detection::BodyType::WORLD_OBJECT, obj1_shapes, obj1_poses,
-                             obj1_types);
-
+  collision_detection_bullet::CollisionObjectWrapperPtr cow_1(new collision_detection_bullet::CollisionObjectWrapper(
```

Comment:
Minor nit: I wouldn't mind a `using namespace collision_detection_bullet;` inside the scope of these functions. Prevents really long lines like this one.

Reply:
sounds reasonable, is there some common policy when to do this and when not?

Reply:
I wouldn't put a `using namespace` into the scope of functions.
Rather, I suggest to have a global
`namespace cb = collision_detection_bullet;`
in the beginning of the cpp file.

Reply:
What are the advantages of that?

Reply:
This way, you can use the short name `cb` all over the cpp file. It's common practice within the MoveIt source:
```
./moveit_ros/perception/mesh_filter/src/depth_self_filter_nodelet.cpp:namespace enc = sensor_msgs::image_encodings;
./moveit_ros/planning_interface/robot_interface/src/wrap_python_robot_interface.cpp:namespace bp = boost::python;
./moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_param_widget.cpp:namespace mpi = moveit::planning_interface;
./moveit_setup_assistant/src/collisions_updater.cpp:namespace po = boost::program_options;
```

Reply:
ok then I'll go with that. Thanks for the hint.

Reply:
That works too, wasn't sure if MoveIt was okay with that though.

Reply:
Not exactly on-topic because I'm lost in the size of the reviews, but did we decide to remove the farm language (cow) and just write out the full name or not?

Reply:
I thought that the consensus was that I remove it as a type (which I did) and still can use the farm language as a variable name :)